packages/service/tests/unit/test_utility_methods.py

```diff
@@ -0,0 +1,35 @@
+import numpy
+import numpy.typing
+
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types import (
+    array_pb2,
+)
+from tests.utilities import _array_utils
+
+
+def test___double2darray_to_ndarray():
```

Comment:
https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions?anchor=facts-and-theories

Name the test using given/when/then clauses and split the test body into arrange/act/assert sections with one blank line between them.

```
def test___valid_double2darray___double2darray_to_ndarray___converts_data():
    double2darray = array_pb2.Double2DArray(data=[1.0, 2.0, 3.0, 4.0, 5.0, 6.0], rows=2, columns=3)

    ndarray = _array_utils.double2darray_to_ndarray(double2darray)

    assert numpy.array_equal(ndarray, numpy.array([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]))